zerver/webhooks/honeybadger/doc.md

```diff
@@ -0,0 +1,23 @@
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Honeybadger will use the `Honeybadger` stream by default if no stream is given
+in the URL. Make sure the selected stream exists in Zulip.
+
+**Follow these steps:**
+
+1. Go to `Alerts & Integrations` page of your `Honeybadger` Console.
+
+2. Find and select the `WebHook` integration option.
+
+3. Enter the URL constructed above into the URL field and configure
+the triggers for the webhook.
+
+4. Test the webhook by clicking `Test This Integration`.
+
+5. Click `Save changes` to create the webhook.
+
```

Comment:
@ViRu-ThE-ViRuS: This looks great! But you are supposed to have have screenshots for the various steps to make it more clear as to where the user is supposed to go/click, etc. I would recommend looking at how the other integrations are documented and adding similar screenshots for Honeybadger. Thanks! :)

Reply:
Added some screenshots (but no so many that they become distracting)!